recipes/libcf/build.sh

```diff
@@ -0,0 +1,15 @@
+export CFLAGS="-Wall -g -m64 -pipe -O2  -fPIC"
+export CXXLAGS="${CFLAGS}"
+export CPPFLAGS="-I${PREFIX}/include"
+export LDFLAGS="-L${PREFIX}/lib"
+export LFLAGS="-fPIC"
+
+CONDA_LST=`conda list`
+if [[ ${CONDA_LST}'y' == *'openmpi'* ]]; then
```

Comment:
Ideally this should not be here. I believe @msarahan and @minrk are working on a way to track mpi variants with features and they can probably chime with the best practice for doing this.

For now I recommend you to choose one and avoid this if-clause.